tigerpath/views.py

```diff
@@ -242,6 +246,11 @@ def update_schedule(request):
 def get_schedule(request):
     curr_user = request.user.profile
     schedule = populate_user_schedule(curr_user.user_schedule)
+
+    # make sure that the schedule has 9 semesters
+    if len(schedule) < 9:
```

Comment:
```suggestion
    while len(schedule) < 9:
```
I don't know if this is needed right now, since all users will presently have either 8 or 9, but it would cover any future changes in schedule length.

Reply:
Like, suppose we expand to 10, and some user still has 8.